webapp/portlet/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -302,6 +302,14 @@
     <skin-name>Enterprise</skin-name>
     <css-path>/skin/css/standalone-pages/AccountSetup.css</css-path>
   </portlet-skin>
+
+  <portal-skin>
+    <skin-name>Enterprise</skin-name>
+    <skin-module>administrationSite</skin-module>
```

Comment:
Define it as portlet skin to avoid displaying the admin style for regular users all time. (waste of bandwidth & browser CSS computing)